### ISSUE TITLE
Headers not passed in, in possibly multiple cases, preventing connected account interaction

### ISSUE THREAD

**Issue by natdm** [stripe/stripe-go#466]
In the most current version of the Go library, the headers are not passed in to the requests.

As an example:
https://github.com/stripe/stripe-go/blob/master/product/client.go#L24
The `params *stripe.ProductParams` that are passed in has a `params.Params` that has a field, `StripeAccount`. This is needed to pass in as a header to signify you are dealing with connected accounts.

It looks like a struct is created here:
https://github.com/stripe/stripe-go/blob/master/product/client.go#L24

and passed in here:
https://github.com/stripe/stripe-go/blob/master/product/client.go#L32

If the code on line 26 (https://github.com/stripe/stripe-go/blob/master/product/client.go#L26-L29)
was made to utilize these parameters by doing 

```go
        var commonParams *stripe.Params
        
	if params != nil {
		body = &form.Values{}
		form.AppendTo(body, params)
                commonParams = &params.Params
	}
```

or something similar, they would actually get passed in.

**Comment by natdm** [comment]
Looks like the sku client doesn't do it either?
https://github.com/stripe/stripe-go/blob/master/sku/client.go#L22-L35

**Comment by natdm** [comment]
Thanks @remi-stripe. Do you guys allow PR's to the codebase at all?

**Comment by remi-stripe** [comment]
@natdm yes we definitely do so if you ever spot anything else wrong, submitting a PR would work. Fixing that one was really easy since you had already found the exact issue :)

**Comment by brandur-stripe** [comment]
Fixed released as 28.0.0.
